### ISSUE TITLE
[BUG] Duplicate/clashing ports used in FVT

### ISSUE THREAD

**Issue by planetf1** [odpi/egeria#6450]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Some FVTs are failing.

One possible cause is that the same ports are used for different tests. This only affects gradle by default, since tests are run in parallel.

### Expected Behavior

No duplicate ports should be used

### Steps To Reproduce

See https://github.com/odpi/egeria/actions/runs/2232481822
and https://github.com/odpi/egeria/actions/runs/2227325114

This may not be the cause of those failures, but it is a latent problem noticed during inspection and should be fixed

### Environment

```markdown
- Egeria:
- OS:
- Java:
- Browser (for UI issues):
- Additional connectors and integration:
```

**Comment by planetf1** [comment]
There are far more clashes in the poms -- many are using 10446.

Will fix those too...

**Comment by planetf1** [comment]
I found quite a few discrepancies - I wonder how the gradle FVTs ever worked as some of the modules recently added were all over the place including discovery & community. This could quite easily lead to a server 'going away' whilst a test was running (since that other test has finished) or possibly assets not being found in an expected state due to other concurrent activity. The intent of the tests was always to keep the isolation between them.

For now I've ensured they are all consistent and unique, and added a top level README with the current allocations. 

Can't be sure this is the cause of intermittent failures, but it's quite possible, and needed fixing anyway

Some refactoring (already done in glossary author & data engine) could help clarify- at least reducing duplication within a module, but even better would be to
 * Dynamically figure out a free port to use
 * Prevent overlapping (could be a gradle constraint easily)
 
 however I've not done further refactoring at this point
